packages/docs-website/docs/docs/server-app-devs/make-api-calls.md

```diff
@@ -0,0 +1,82 @@
+---
+id: make-api-calls
+title: Make Wallet API calls
+---
+
+import Mermaid from '@theme/Mermaid';
+
+:::note
+This page relates to the statechannels server wallet ONLY.
+:::
+
+In the following diagram A and B are each a separate server application each with their own statechannels server wallet instance running in the same process.
+
+The diagram shows a typical "happy path", directly-funded state channel interaction between those applications, their respective Wallets, and the Blockchain. The id of the channel is '0xabc', for illustrative purposes.
+
+<Mermaid chart="
+sequenceDiagram
+participant WalletA
+participant A
+participant B
+participant WalletB
+rect rgba(0, 0, 255, .1)
+note left of WalletB: Opening a channel
+A->>WalletA: createChannel();
+WalletA-->>A: 0xabc: opening
+WalletA-->>A: outbox: [msg0]
+A->>B: msg0
+B->>WalletB: pushMessage(msg0)
+WalletB-->>B: 0xabc: proposed
+B->WalletB: joinChannel('0xabc');
+WalletB-->>B: 0xabc opening
+WalletB-->>B: outbox: [msg1]
+B->>A: msg1;
+A->>WalletA: pushMessage(msg1);
+WalletA-->>A: 0xabc: funding
+WalletA->>Chain: deposit()
+Chain-->>WalletA: Deposited
+Chain-->>WalletB: Deposited
+WalletB->>Chain: deposit()
+Chain-->>WalletA: Deposited
+Chain-->>WalletB: Deposited
+WalletA-->>A: outbox: [msg2]
+A->>B: msg2
+B->>WalletB: pushMessage(msg2)
+WalletB-->>B: 0xabc: running
+WalletB-->>B: MessageQueued(msg3)
+B->>A: msg3;
+A->>WalletA: pushMessage(msg3);
+WalletA-->>A: 0xabc: running
+end
+loop 0...m
+note left of WalletB: Running a channel
+A->>WalletA: updateChannel(state-4+2m);
```

Comment:
I don't really understand the math here.
- Is it meant to be `4+2i`? When I view the diagram, it suggests that we update the channel `m` times, so there should probably be a loop counter variable `i` or something.
- If ^ this is true, maybe the math is worth clarifying, something like
```suggestion
A->>WalletA: updateChannel(state-(2i+3)+1);
```